src/Pate/Verification/AbstractDomain.hs

```diff
@@ -392,16 +394,25 @@ applyAbsRange sym e rng = case rng of
 absDomainValToAsm ::
   W4.IsSymExprBuilder sym =>
   sym ->
+  PE.EquivContext sym arch ->
   PSR.MacawRegEntry sym tp ->
+  Maybe (MAS.AbsValue (MM.ArchAddrWidth arch) tp) ->
   MacawAbstractValue sym tp ->
```

Comment:
There are now two abstract values here - can you add some commentary on the difference between the two?

Reply:
Added comment